src/v2/ion/i18nTransformer.js

```diff
@@ -65,6 +65,7 @@ const I18N_OVERRIDE_MAPPINGS = {
   'select-authenticator-enroll.authenticator.okta_verify': 'oie.okta_verify.label',
   'select-authenticator-enroll.authenticator.google_authenticator': 'oie.google_authenticator.label',
   'select-authenticator-enroll.authenticator.rsa_token': 'oie.rsa.label',
+  'select-authenticator-enroll.authenticator.custom_otp': 'oie.verify.custom_otp.title',
```

Comment:
you also need to map the string for the challenge field

Reply:
This also requires you to add an entry in login.properties for `oie.custom_otp.verify.passcode.label = Enter code`